### DISCUSSION TITLE
[Documentation] Need a feature table what is available in Ms.Id.Web (ASP.NET Core and other) and MSAL.NET

### DISCUSSION THREAD

**Discussion by jmprieur** [AzureAD/microsoft-identity-web#1068] [Documentation Gaps]
### Why?
Microsoft Identity Web is the recommended solution for ASP.NET Core web apps, web APIs, Azure functions and gRPC services secured with the Microsoft.Identity.Platform.

It also provides some limited support for .NET Core and .NET Framework (including ASP.NET), where customers still need to use a hybrid solution with MSAL.Net

Customers need to quickly be able to choose a solution and a first step is the decision tree in [Is MSAL.NET right for me?](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Is-MSAL.NET-right-for-me%3F), however, there are still questions such as, is Pop supported in Microsoft identity web.

### Please check all that apply

- [ x ] documentation doesn't exist

### Proposed solution 

Have a table showing what features are available in:
- Microsoft.Identity.Web for ASP.NET Core
- Microsoft.Identity.Web for ASP.NET, .NET Framework, .Net Core
- MSAL.NET

possibly with a hyperlink on how to do.

**Comment by jennyf19** [comment]
Have started this [wiki page](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Is-MSAL.NET-right-for-me%3F) in MSAL.NET - Is MSAL.NET right for me?
